Client/Clique/Modules/Profile/ProfileViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  Clique
+//  Created by Infinum on 15.11.2022..
+//
+
+import UIKit
+import NVActivityIndicatorView
+import IQKeyboardManagerSwift
+import SkeletonView
+
+class ProfileViewController: UIViewController {
+    
+    @IBOutlet private var profileImage: UIImageView!
+    @IBOutlet private var myEventsButton: UIButton!
+    @IBOutlet private var eventsButtons: [UIButton]!
+    @IBOutlet private var joinedEventsButton: UIButton!
+    @IBOutlet private var pastJoinedEventsButton: UIButton!
+    @IBOutlet private var createdEventsButton: UIButton!
+    @IBOutlet private var pastCreatedEventsButton: UIButton!
+    @IBOutlet private var labelProfileName: UILabel!
+    @IBOutlet private var bioTextView: UITextView!
+    
+    private let profileService = ProfileService()
+    
+//    override func viewDidAppear(_ animated: Bool) {
+//        super.viewDidAppear(animated)
+//        profileImage.circleImage()
+//        getUser()
+//        profileImage.layer.masksToBounds = false
+//        labelProfileName.layer.masksToBounds = false
+//        textViewBio.layer.masksToBounds = false
+//
+//        profileImage.isSkeletonable = true
+//        profileImage.showAnimatedSkeleton(usingColor: .clouds,
+//                                          transition: .crossDissolve(0.5))
+//
+//        labelProfileName.isSkeletonable = true
+//        labelProfileName.showAnimatedSkeleton(usingColor: .clouds,
+//                                              transition: .crossDissolve(0.5))
+//
+//        textViewBio.isSkeletonable = true
+//        textViewBio.showAnimatedSkeleton(usingColor: .clouds,
+//                                         transition: .crossDissolve(0.5))
+//    }
+//    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        getUser()
+<<<<<<< HEAD
+=======
```

Comment:
In this file, you have a conflict, so please check this out and resolve it.
If you don't do that, the project won't let you build the app.